kong/plugins/file-log/handler.lua

```diff
@@ -63,13 +66,9 @@ FileLogHandler.PRIORITY = 9
 FileLogHandler.VERSION = "2.0.0"
 
 function FileLogHandler:log(conf)
-  local message = basic_serializer.serialize(ngx)
-
-  local ok, err = ngx_timer(0, log, conf, message)
-  if not ok then
-    ngx.log(ngx.ERR, "[file-log] failed to create timer: ", err)
-  end
+  local message = serialize(ngx)
```

Comment:
We should bump the plugin version as well (just above this line)

Reply:
Thanks, fixed!